apps/services/auth-admin-api/src/buildOpenApi.ts

```diff
@@ -0,0 +1,10 @@
+import { buildOpenApi } from '@island.is/infra-nest-server'
+
+import { AppModule } from './app/app.module'
+import { openApi } from './openApi'
+
+buildOpenApi({
+  path: 'apps/services/auth-admin-api/src/openapi.yaml',
```

Comment:
`path.join(__dirname__, 'openapi.yaml')`?

Reply:
Good point, I will keep that for another PR. I would like to merge the PR to avoid the conflict loop it can get stuck in